src/app/app.component.ts

```diff
@@ -20,8 +20,9 @@ export class App {
     private authenticationProvider: AuthenticationProvider,
   ) {
     platform.ready().then(() => {
-      statusBar.styleDefault();
+      statusBar.styleLightContent();
       statusBar.overlaysWebView(false);
+      statusBar.backgroundColorByHexString('#000000');
```

Comment:
Should this be part of the SCSS, and maybe using the existing GDS colours?

Reply:
I spent a bit of time looking for a solution in SCSS but couldn't find one, probably because this interface is through a Cordova plugin, not Ionic itself. I'll have a bit more of a read around for a better solution.

Reply:
I see, that's fine then